web-ui/src/main/resources/catalog/templates/admin/settings/system.html

```diff
@@ -234,6 +234,24 @@ <h3>{{section2.name | translate}}</h3>
                           {{'Administrator' | translate}}
                         </option>
                       </select>
+
+                      <select data-ng-switch-when="metadata/batchediting/accesslevel"
+                              class="form-control"
+                              name="{{s.name}}">
+                        <option value=""
```

Comment:
No need for null value as it is editor before this new option was added so it should be editor by default.  It may not be clear what Null/empty value mean?